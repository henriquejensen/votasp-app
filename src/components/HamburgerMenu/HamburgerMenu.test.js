import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import HamburgerMenu from './HamburgerMenu';

describe('HamburgerMenu', () => {
  it('should match a snapshot', () => {
    const onClick = jest.fn();
    const Component = shallow(( <HamburgerMenu onClick={onClick} /> ));
    expect(toJson(Component)).toMatchSnapshot();
    Component.props().onClick();
    expect(onClick).toHaveBeenCalled();
  })
})
