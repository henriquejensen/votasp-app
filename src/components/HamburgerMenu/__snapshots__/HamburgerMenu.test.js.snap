// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HamburgerMenu should match a snapshot 1`] = `
<button
  className="hamburger-menu"
  onClick={[MockFunction]}
  type="button"
>
  <svg
    height="25px"
    viewBox="0 0 22.67 22.82"
    width="25px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>
      Hamburguer Menu
    </title>
    <g>
      <path
        className="listraHamburguer"
        d="M20.94,0H1.7A1.71,1.71,0,0,0,0,1.7H0A1.72,1.72,0,0,0,1.7,3.4H21a1.72,1.72,0,0,0,1.7-1.7h0A1.74,1.74,0,0,0,20.94,0Z"
      />
      <path
        className="listraHamburguer"
        d="M20.94,19.42H1.7A1.72,1.72,0,0,0,0,21.12H0a1.71,1.71,0,0,0,1.7,1.7H21a1.72,1.72,0,0,0,1.7-1.7h0A1.75,1.75,0,0,0,20.94,19.42Z"
      />
      <path
        className="listraHamburguer"
        d="M20.94,9.73H1.7A1.71,1.71,0,0,0,0,11.43H0a1.71,1.71,0,0,0,1.7,1.69H21a1.72,1.72,0,0,0,1.7-1.69h0A1.74,1.74,0,0,0,20.94,9.73Z"
      />
    </g>
  </svg>
</button>
`;
